code/game/objects/items/grenades/plastic.dm

```diff
@@ -108,8 +108,9 @@
 		return
 
 	to_chat(user, "<span class='notice'>You start planting [src]. The timer is set to [det_time]...</span>")
+	to_chat(target, "<span class='notice'>[name] is planting a bomb on you!.</span>")
```

Comment:
[name] is not the name name of the bomber, neither is target the one being bombed